app/models/tweet.rb

```diff
@@ -0,0 +1,9 @@
+class Tweet < ApplicationRecord
+  belongs_to :author
+  belongs_to :bookmark
+  belongs_to :like
```

Comment:
this is wrong the like belongs to the tweet, and in the tweet it should have has_many likes